test/test_mod_taken_together_queries.py

```diff
@@ -60,54 +66,70 @@ def test_query_module_taken_together_specified_mod_specified_aysem(self):
             with specified module in the specified semester
         '''
 
+        # Add some dummy students and modules
+        model.add_student('dummyYr1A', 1)
+        model.add_module('AAA1111', 'Dummy 1', 'Description', 4, 'Active')
+        model.add_module('AAA1112', 'Dummy 2', 'Description', 4, 'Active')
+        model.add_student_plan('dummyYr1A', True, 'AAA1111', 'AY 17/18 Sem 2')
+        model.add_student_plan('dummyYr1A', True, 'AAA1112', 'AY 17/18 Sem 2')
+
         list_of_mod_taken_together = \
-            model.get_mod_taken_together_with_mod_and_aysem('CS2105', 'AY 16/17 Sem 1')
+            model.get_mod_taken_together_with_mod_and_aysem('AAA1111', 'AY 17/18 Sem 2')
 
-        required_list = [('CS2105', 'Introduction to Computer Networks', 'CS1231',
-                          'Discrete Structures', 2),
-                         ('CS2105', 'Introduction to Computer Networks', 'CS1010',
-                          'Programming Methodology', 2)]
+        required_list = [('AAA1111', 'Dummy 1', 'AAA1112', 'Dummy 2', 1)]
 
         assert_equal(len(list_of_mod_taken_together), len(required_list))
         assert_equal(sorted(list_of_mod_taken_together), sorted(required_list))
         assert_true(self.is_count_in_non_ascending_order(list_of_mod_taken_together))
 
         list_of_mod_taken_together = \
```

Comment:
This test case should be extracted out into another separate one. Not recommended to test different cases together in one test case.